config/application.rb

```diff
@@ -27,5 +27,7 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+
+    config.action_mailer.default_url_options = { host: "localhost:3000" }
```

Comment:
Can you also add this to the production config in the `environments` directory. In production, we should use ENV['HOST'].